examples/sslecho/test.d

```diff
@@ -0,0 +1,25 @@
+/// Runs the server and the client in a fork for CI
+module test;
+
+import core.thread;
+import core.time;
+
+import std.process;
+import std.stdio;
+
+int main ()
+{
+    auto serverPid = spawnProcess([ "./sslecho", "s" ]);
+    writeln("Server has been spawned");
+    Thread.sleep(3.seconds);
+    auto clientPid = spawnShell(`echo "Hello World\nkill" | ./sslecho c localhost`);
```

Comment:
Does `\n` here print a newline? I don't understand what would make it do that here. Either `echo` needs `-e`, or using `printf` instead of `echo`, or not using D WYSIWYG strings would make it print a newline.

Reply:
That might explain why it was not working in the shell. I would be more than happy if you could take a look at improving it. I have quite a few bug fixes to upstream ATM. I'll also see if I can port more examples, but I don't have as much experience with OpenSSL as you.